src/textual/app.py

```diff
@@ -1275,22 +1263,42 @@ def bell(self) -> None:
         if not self.is_headless:
             self.console.bell()
 
-    async def check_bindings(self, key: str) -> bool:
+    @property
+    def _binding_chain(self) -> list[tuple[DOMNode, Bindings]]:
+        """Get a chain of nodes and bindings to consider. Goes from focused widget to app, or
+        screen to app.
```

Comment:
To me "goes from focused widget to app" didn't make it clear that it goes via all the ancestors.

```suggestion
        """Get a chain of nodes and bindings to consider. If no widget is focused, returns the bindings from both the screen and the app level bindings. Otherwise, combines all the bindings from the currently focused node up the DOM to the root App.
```